content/300-guides/100-performance-and-optimization/090-metrics.mdx

```diff
@@ -0,0 +1,411 @@
+        previousHistograms[histogramIndex].value.buckets[bucketIndex]
+      let change = count - prev
+
+      for (let sendTimes = 0; sendTimes < change; sendTimes++) {
+        statsd.timing('prisma.' + diffHistograms.key, bucket)
+      }
+    })
+  })
+
+  previousHistograms = diffHistograms
+}
+
+setInterval(async () => await statsdSender(), 10000)
+```
+
+## Get metrics in Prometheus format
+
+When you get Prisma metrics in Prometheus format, you can use them as returned, or [send them to the Prometheus metrics system](#use-prisma-metrics-with-the-prometheus-metrics-system).
+
+To get Metrics in a Prometheus format, add the following lines to your application code:
+
+```ts
+let metrics = prisma.$metrics.prometheus()
+console.log(metrics)
+```
+
+This returns metrics as follows:
+
+```c
+# HELP query_total_operations The total number of operations executed
+
+# TYPE query_total_operations counter
+
+query_total_operations 0
+
+# HELP query_total_queries The total number of queries executed
+
+# TYPE query_total_queries counter
+
+query_total_queries 0
+
+# HELP pool_active_connections The number of active connections in use
+
+# TYPE pool_active_connections gauge
+
+pool_active_connections 0
+
+# HELP pool_idle_connections The number of connections that are not being used
+
+# TYPE pool_idle_connections gauge
+
+pool_idle_connections 21
+
+# HELP pool_wait_count The number of workers waiting for a connection
+
+# TYPE pool_wait_count gauge
+
+pool_wait_count
+
+# HELP query_active_transactions The number of active transactions
+
+# TYPE query_active_transactions gauge
+
+query_active_transactions
+
+# HELP pool_wait_duration_ms The wait time for a worker to get a connection.
+
+# TYPE pool_wait_duration_ms histogram
+
+pool_wait_duration_ms_bucket{le="0"} 0
+pool_wait_duration_ms_bucket{le="1"} 1
+pool_wait_duration_ms_bucket{le="5"} 1
+pool_wait_duration_ms_bucket{le="10"} 1
+pool_wait_duration_ms_bucket{le="50"} 1
+pool_wait_duration_ms_bucket{le="100"} 1
+pool_wait_duration_ms_bucket{le="500"} 1
+pool_wait_duration_ms_bucket{le="1000"} 1
+pool_wait_duration_ms_bucket{le="5000"} 1
+pool_wait_duration_ms_bucket{le="50000"} 1
+pool_wait_duration_ms_bucket{le="+Inf"} 1
+pool_wait_duration_ms_sum 0.050875000000000004
+pool_wait_duration_ms_count 1
+```
+
+### Use Prisma metrics with the Prometheus metrics system
+
+In the majority of cases, Prometheus must scrape an endpoint to get metrics. The following example shows how to send data with `Express.js`:
+
+```javascript
+import { PrismaClient } from "@prisma/client";
+import express, { Request, Response } from "express";
+
+const app = express();
+const port = 4000;
+const prisma = new PrismaClient();
+
+app.get("/metrics", async (\_req, res: Response) => {
+let metrics = await prisma.$metrics.prometheus();
+res.end(metrics);
+});
+
+app.listen(port, () => {
+console.log(`Example app listening on port ${port}`);
+})
+
+Combining it with another Prometheus client library:
```

Comment:
Can you just check how this will render? I think you need to end the code block on 308 and start a new code block on 310